task_manager/templates/base.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+{% load i18n %}
+<html lang="">
+  <head>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>{% trans "Task manager Hexlet" %}</title>
+    {% load django_bootstrap5 %} {% bootstrap_css %} {% bootstrap_javascript %}
```

Comment:
[optional] wdyt, is it the right time to make `prettier` jinja-aware? [this](https://github.com/davidodenwald/prettier-plugin-jinja-template) might help